terraform/aks/application.tf

```diff
@@ -1,6 +1,7 @@
 locals {
   environment  = "${var.app_environment}${var.app_suffix}"
```

Comment:
Not one fornow but I think it'd be clearer if we added the `-` to the suffix here instead of when doing `F_VAR_app_suffix=-$(PULL_REQUEST_NUMBER)`, it's easily lost in the assignment